testsuite/testcases/src/load_vs_perf_benchmark.rs

```diff
@@ -67,9 +79,16 @@ impl Workloads {
             Self::TRANSACTIONS(workloads) => workloads[index].configure(request),
         }
     }
+
+    fn split_duration(&self, global_duration: Duration) -> (Duration, Duration) {
+        let total_phases: usize = (0..self.len()).into_iter().map(|index| self.num_phases(index)).sum();
+        let phase_duration = global_duration.div_f32(total_phases as f32 + 0.2 * (self.len() - 1) as f32);
```

Comment:
nit: might be worth making `0.2` a const, to make it obvious that it's the duration buffer we're using? 😄 That way, you can also reference it in tests.